test/utility/BlockNumber.ts

```diff
@@ -0,0 +1,17 @@
+import Contracts, { TestBlockNumber } from '../../components/Contracts';
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+
+describe('BlockNumber', () => {
+    let blockNumber: TestBlockNumber;
+
+    beforeEach(async () => {
+        blockNumber = await Contracts.TestBlockNumber.deploy();
+    });
+
+    it('should return the time of the current block number', async () => {
+        expect(await blockNumber.callStatic.realBlockNumber()).to.equal(
```

Comment:
Why is `callStatic` needed here?

Reply:
Because each non-call transaction increments the current block number/time